Gemfile

```diff
@@ -9,6 +9,7 @@ gem "haml", "~> 5.0", ">= 5.0.1"
 gem "nokogiri", "~> 1.8"
 gem "haml_lint"
 gem "scss_lint"
```

Comment:
Lets remove `haml_lint` and `scss_lint` from the Gemfile. They should be installed locally but don't need to be a dependency of the project. Plus, they get updated a lot and Gemnasium complains. 😉